package.json

```diff
@@ -105,5 +104,9 @@
       "last 1 safari version"
     ]
   },
-  "license": "GPL-3.0-or-later"
+  "license": "GPL-3.0-or-later",
+  "dependencies": {
+    "fuse-default-token-list": "^1.5.4",
+    "uniswap-fuse-sdk": "^3.0.3-a4"
```

Comment:
can you add link to this repo?

Reply:
here is the repo for the default token list https://github.com/mul53/default-token-list

Reply:
was looking for a repo for uniswap-fuse-sdk :)

Reply:
here it is https://github.com/mul53/uniswap-sdk

Reply:
thanks 👍 
can you move the dependencies to devDependecnies like uniswap team is doing?

Reply:
Done